### PR TITLE
metabase: use jdk11

### PR DESCRIPTION
**Pull request by raboof**
###### Motivation for this change

jdk15 doesn't work, https://github.com/metabase/metabase/issues/13881

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Please replace the jre in metabase with jdk11 instead.

**Comment by raboof** [comment]
> Please replace the jre in metabase with jdk11 instead.

While I see some advantage of specifying the version inside the package, there is also an advantage of specifying it in all-packages.nix.

There are many other packages that specify the jre/jdk version like this in all-packages.nix, and also when this topic was recently discussed (https://discourse.nixos.org/t/package-version-arguments-vs-generic-package-arguments/9071) there was no clear preference.

I'll make the change, but I don't like adding extra round-trips to PR's like this. It discourages contributions without good reason IMHO.

**Comment by SuperSandro2000** [comment]
> > Please replace the jre in metabase with jdk11 instead.
> 
> While I see some advantage of specifying the version inside the package, there is also an advantage of specifying it in all-packages.nix.
> 
> There are many other packages that specify the jre/jdk version like this in all-packages.nix, and also when this topic was recently discussed ([discourse.nixos.org/t/package-version-arguments-vs-generic-package-arguments/9071](https://discourse.nixos.org/t/package-version-arguments-vs-generic-package-arguments/9071)) there was no clear preference.
> 
> I'll make the change, but I don't like adding extra round-trips to PR's like this. It discourages contributions without good reason IMHO.

Good to know that and good reason. If we follow that discussion we would need to replace jre with jdk. My main problem was with replacing jre with jdk which is not intuitiv.

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109832` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>metabase</li>
</ul>
</details>
